library/common/config/config.cc

```diff
@@ -89,6 +89,15 @@ const std::string config_header = R"(
   typed_config:
     "@type": type.googleapis.com/envoy.extensions.transport_sockets.tls.v3.UpstreamTlsContext
     common_tls_context:
+      tls_params:
+        tls_maximum_protocol_version: TLSv1_3
+        cipher_suites: ["ALL:!aPSK:!ECDSA+SHA1:!3DES"]
```

Comment:
This should be the BoringSSL with the same modifications that Chrome uses (https://source.chromium.org/chromium/chromium/src/+/main:net/socket/ssl_client_socket_impl.cc;drc=c361f8a4b30a6b9f289107b8085d63876114dba5;l=854)

assuming that 3DES is not enabled and RSA is allowed

Reply:
Yup, this is exactly what I see in my instance of Chrome.

Reply:
Do we need to change the code here, too? https://github.com/envoyproxy/envoy/blob/main/source/extensions/transport_sockets/tls/context_config_impl.cc#L319
I'm not sure I understand the relationship between these two?

Reply:
IMO we probably should change the default, but that is outside of the scope of this PR which is to specifically configure it right now for EM.

Reply:
*facepalm* I completely missed that this PR is an Envoy *Mobile* PR. Ok, that makes a lot more sense. Agreed that it's clearly out of scope for this PR :) (But would be good for an Envoy PR.)

Reply:
We probably need to also discuss GREASE for 1.3, but also outside the scope of this because AIUI that requires additional Envoy code to enable it in bssl.